membrane_curvature/tests/test_mdakit_membcurv.py

```diff
@@ -239,3 +278,14 @@ def test_curvature(md_ref_beads, mdtraj_po4):
 
     for k, k_test in zip(MEMBRANE_CURVATURE_DATA['gaussian_from_z_avg'][lf], K_test[lf]):
         assert_almost_equal(k, k_test)
+
+
+def test_def_all_beads():
+    lipid_types = ['POPC', 'POPE']
+    lfs = ["upper", "lower"]
+    head_list = [4286, 11531, 18891]
+    topology = md.load(GRO_MEMBRANE_PROTEIN).topology
+    beads_test = def_all_beads(lipid_types, lfs, head_list, topology)
+    for lt in lipid_types:
+        for bead, bead_t in zip(MEMBRANE_CURVATURE_DATA['beads']["upper"][lt], beads_test["upper"][lt]):
```

Comment:
What about the lower beads?

Reply:
>Could you please add a smaller example that's more easily human-verifiable (e.g. 10 lipids in each leaflet)? 
Should I add a new test (smaller number of beads), or is it better to replace the current one?

Reply:
A new test in addition to this would be the best of both worlds ^_^